src/docs/build-deploy-and-maintain-apps/maintain-an-application.md

```diff
@@ -0,0 +1,495 @@
+
+Before scheduled changes to the platform and shared services occur, the Platform Services Team will get in touch with all registered Product Owners and Technical Leads through the [Product Registry](https://registry.developer.gov.bc.ca/). To ensure your team receives timely notices about upcoming events, please maintain accurate and up-to-date contact information for POs and TLs on the Product Registry.
+
+Moreover, it's advisable for all team members to remain engaged with the community through the following channels:
+
+- Subscribe to [Platform Services Newsletters](https://subscribe.developer.gov.bc.ca/) so that you will receive email notifications for upcoming changes, as well as [the Platform Community MeetUp Event](https://cloud.gov.bc.ca/private-cloud/support-and-community/events-in-the-bc-gov-private-cloud-paas/) every 3 weeks for platform updates and technical demos.
+
+- Monitor [Rocket.Chat #devops-alerts channel](https://chat.developer.gov.bc.ca/channel/devops-alerts) for service alerts and status updates. Rocket.Chat serves as the main communication method for teams on the platform to get help, share knowledge and information, and build up the community. 
+
+- Take a look at [additional Platform activities and resources](https://cloud.gov.bc.ca/private-cloud/support-and-community/) periodically. This include events for OpenShift training , StackOverflow knowledge base, and more.
+---
+## Managing impacts of personnel changes
+
+During the maintenance stage of an application's lifecycle, the development team frequently undergoes changes. Team members might leave for other positions or be assigned to new projects, and responsibilities get transferred to operational teams. These changes have a significant impact on the OpenShift platform because there are high expectations for the app team to keep their application current and promptly address platform changes.
+
+
+The changes and the resulting loss of expertise have a significant impact on the OpenShift platform. This is because the app team is expected to keep their application up-to-date and respond promptly to changes on the platform. 
+
+### Onboarding New Team Members
+
+When a new member joins your team, they should go through a similar onboarding process as any other person joining the OpenShift platform. Along with getting familiar with the specific application(s) supported by their new team, they should also:
+
+* Attend OpenShift 101 and 201 training
+* Go through the onboarding process which can be done with your team's Product owner or by scheduling a meeting with a member of the Platform team. 
+* Join the community on Rocket.Chat, Stack Overflow and our Platform Community Meetups
+* Be granted access to all relevant namespaces, repositories and other services Remember to also update the Registry, if needed. 
```

Comment:
This section might benefit from links to the services and mentioned, training docs etc.

Reply:
New update - includes all of the links. Thank you Matt!